pyaccel/new_lifetime.py

```diff
@@ -1,7 +1,9 @@
 import pyaccel
 import pymodels
```

Comment:
@AnaLuisaAR, pyaccel is a general purpose library, not specific for Sirius. On the other hand, pymodels is our package with the lattice models of Sirius accelerators. Hence, pyaccel must have pymodels as a dependency.

 It is better to make all the functions implemented here independent of the model, and use all variables which are lattice dependent as input for these functions.